pkg/trace/api/container.go

```diff
@@ -0,0 +1,23 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016 Datadog, Inc.
```

Comment:
2022?

Reply:
(same for container_linux.go)

Reply:
I think this is following the Go tracer 😄 😮‍💨 😞